design/src/rfcs/rfc0027_improve_sdk_request_id_access.md

```diff
@@ -0,0 +1,244 @@
+RFC: Improving access to request IDs in SDK clients
+===================================================
+
+> Status: RFC
+>
+> Applies to: AWS SDK clients
+
+At time of writing, customers can retrieve a request ID in one of four ways in the Rust SDK:
+
+1. For error cases where the response parsed successfully, the request ID can be retrieved
+   via accessor method on operation error. This also works for unmodeled errors so long as
+   the response parsing succeeds.
+2. For error cases where a response was received but parsing fails, the response headers
+   can be retrieved from the raw response on the error, but customers have to manually extract
+   the request ID from those headers (there's no convenient accessor method).
+3. For all error cases where the request ID header was sent in the response, customers can
+   call `SdkError::into_service_error` to transform the `SdkError` into an operation error,
+   which has a `request_id` accessor on it.
```

Comment:
does "sent in the response" mean it can be sent as a header OR in the body?

Reply:
It says, "request ID _header_".